src/resmom/stage_func.c

```diff
@@ -1210,6 +1213,15 @@ rmjobdir(char *jobid, char *jobdir, uid_t uid, gid_t gid)
 	if (jobdir == NULL)
 		return;
 
+	if (check_shared &&
+	    (pbs_jobdir_root[0] != '\0') &&
+	    pbs_jobdir_root_shared &&
+	    (strncmp(pbs_jobdir_root, jobdir, strlen(pbs_jobdir_root)) == 0)) {
+		log_eventf(PBSEVENT_DEBUG3, PBS_EVENTCLASS_JOB, LOG_DEBUG, jobid?jobid:__func__, "shared jobdir %s to be removed by primary mom", jobdir);
+
```

Comment:
I'd say just jobid, not jobid with the function.  That way you'll see this message with tracejob.

Reply:
The function where this is called tolerates a NULL jobid. I don't think that will really happen but just to guard against crash, I'm handling such a case. Instead of '__func__', I'll replace with just the string "jobid".

Reply:
use "", not "jobid"

Reply:
Ok, will use "".